core/core.go

```diff
@@ -68,6 +69,9 @@ func Run() error {
 	// Set up channel to receive Unix Signals.
 	signals(state)
 
+	// Start the Repo Manager instance
+	log.Infof("Starting the Control Workflows repo manager")
+	_ = the.RepoManager()
```

Comment:
@teo is this the correct place to put this?

Reply:
Yes, I think it's a good enough place.
However 6s for startup is quite a bit, I wonder if there could be some low hanging fruit in the !public parsing that allows us to trim it down.

If you feel like going down the optimization rabbit hole, `utils.TimeTrack` and `utils.TimeTrackFunction` can help.

Reply:
I agree that `6s` might be a little bit problematic especially within the Ansible recipe. Other services are dependent on `AliECS Core` being up and running. 

Could we maybe start the server and after that, in the background, Core does the reading and parsing of the files?
I believe there is no `GetWorkflowTemplates` call right at the start of the server.

Reply:
Slow startup addressed by the latest changes.